sample/src/main/res/layout/fragment_editable_list_view.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@style/SampleScrollView">
+
+    <LinearLayout
+        android:id="@+id/content_layout"
+        style="@style/SampleSectionContainer">
+
+        <TextView style="@style/PlaceholderText" />
+
+        <FrameLayout style="@style/PlaceholderContainer">
+
+            <uk.gov.hmrc.components.organism.editable.EditableListView
+                android:id="@+id/editable_list_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent" />
+        </FrameLayout>
+
+        <TextView style="@style/ExampleText" />
+
+        <com.google.android.material.card.MaterialCardView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/hmrc_spacing_16"
```

Comment:
Can we add `android:paddingTop="@dimen/hmrc_spacing_16` to these example cards to make them look a bit more like they will when used in the app?